egs/librispeech/asr1/local/run_spm.sh

```diff
@@ -98,81 +124,97 @@ if [ ${stage} -le 1 ]; then
     echo "stage 1: Feature Generation"
     fbankdir=fbank
     # Generate the fbank features; by default 80-dimensional fbanks with pitch on each frame
-    for x in train_si284 test_dev93 test_eval92; do
-        steps/make_fbank_pitch.sh --cmd "$train_cmd" --nj 10 data/${x} exp/make_fbank/${x} ${fbankdir}
+    for x in dev_clean test_clean dev_other test_other train_clean_100 train_clean_360 train_other_500; do
+        steps/make_fbank_pitch.sh --cmd "$train_cmd" --nj 32 data/${x} exp/make_fbank/${x} ${fbankdir}
     done
 
+    utils/combine_data.sh data/${train_set} data/train_clean_100 data/train_clean_360 data/train_other_500
```

Comment:
Can you check https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/run.sh#L130 ?
Now we use `remove_longshortdata.sh` to remove long sentences.